internal/display/apps.go

```diff
@@ -305,3 +305,22 @@ func interfaceSliceToString(s []interface{}) []string {
 	}
 	return res
 }
+
+func applyColor(a string) string {
```

Comment:
do you think we can unify the _friendly_ names on a single place as constants?
It is also hardcoded here: https://github.com/auth0/auth0-cli/blob/71463818fa582c1ebdca035c2d4d922b1d958da9/internal/display/apps.go#L266-L279

Reply:
yea good call 👍